JapaneseFilter/sections/specific.txt

```diff
@@ -60,6 +60,9 @@ gundamlog.com###more > iframe[src^="https://richlink.blogsys.jp"]
 !
 !
 !
+enterprisezine.jp##.c-article_sns
+enterprisezine.jp##.c-article_function_sns
+enterprisezine.jp##button[class="c-article_popup_btn"]
```

Comment:
Please move to `SocialFilters`.

Reply:
That was my question above. It turns out that their purpose is more important than their language affiliation, thank you!

Reply:
I meant everything that you made change.

Reply:
I moved one rule to popups (which blocks it), the other two to specific